_health-care/_js/actions/index.js

```diff
@@ -0,0 +1,17 @@
+export const ENSURE_FIELDS_INITIALIZED = 'ENSURE_FIELDS_INITIALIZED';
+export const VETERAN_FIELD_UPDATE = 'VETERAN_FIELD_UPDATE';
+
+export function ensureFieldsInitialized(path) {
+  return {
+    type: ENSURE_FIELDS_INITIALIZED,
+    path
+  };
+}
+
+export function updateField(propertyPath, value) {
+  return {
+    type: VETERAN_FIELD_UPDATE,
```

Comment:
not really. I originally had this in a veterans file so it looked redundant, but let me change.